test/services/MailService.spec.ts

```diff
@@ -0,0 +1,56 @@
+import MailService from '@/services/MailService';
+import mail from '@test/data/mail';
+import message from '@test/data/message';
+import { expect } from 'chai';
+import 'mocha';
+import nodeMailer from 'nodemailer';
+
+describe('MailService.send', () => {
+
+  // Instantiate the service
+  const mailService = new MailService();
+  let messageInfo: any;
+
+  before('should send an e-mail.', async () => {
+    // Create ethereal mail account
```

Comment:
ethereal? 🙃

Reply:
Is a test service made for nodemailer. https://ethereal.email/